ansible/group_vars/all.yml

```diff
@@ -15,3 +15,12 @@ rhacm_redisgraph_memory_limit: "3Gi"
 hub_cluster_kubeconfig: /home/stack/ocp_clusters/vlan608/auth/kubeconfig
 managed_cluster_name: shiftstack0000
 managed_cluster_kubeconfig: "/home/stack/ocp_clusters/{{managed_cluster_name}}/auth/kubeconfig"
+
+################################################################################
+# Local-storage vars:
+################################################################################
+shiftstack_cluster: vlan608
+local_storage_instance_type: nvme-8
+local_storage_device: /dev/nvme0n1
```

Comment:
Alex I see you have tailored this to your use case. But here's some thoughts as multiple places in webfuse use local storage
1. For logging
2. For prometheus to be backed by persistent storage.

Wondering if we can make this generic enough for all usecases

Reply:
Got it, I'll look into exposing more vars so other use cases can consume this too.